cartographer/mapping/internal/constraints/constraint_builder_3d.cc

```diff
@@ -224,18 +229,25 @@ void ConstraintBuilder3D::ComputeConstraint(
       kGlobalConstraintLowResolutionScoresMetric->Observe(
           match_result->low_resolution_score);
     } else {
+      LOG(INFO) << "[DROPPED] " << search_type << ": Dropped from "
+          << node_id.trajectory_id << ", node " << node_id.node_index
+          << " to " << submap_id.submap_index;
       return;
     }
   } else {
-    bool mapping = submap_id.trajectory_id == 0 && node_id.trajectory_id == 0;
-    bool initializing = submap_id.submap_index <= 1 && node_id.trajectory_id == 0;
-    bool log = mapping || initializing || submap_id.trajectory_id != node_id.trajectory_id;
+    const bool mapping = submap_id.trajectory_id == 0 && node_id.trajectory_id == 0;
```

Comment:
why is this called mapping? looks like something else to me..

Reply:
Searching constraints from `trajectory_id == 0` to `trajectory_id == 0`.
-> No other trajectory before that, so it is mapping 

What does it look like to you?

Reply:
can't it also be mapping if `trajectory_id  == 100`? wouldn't it be more related to `submap_id.trajectory_id == node_id.trajectory_id`? or maybe I fully misunderstand..

Reply:
`submap_id.trajectory_id == node_id.trajectory_id` is inter trajectory constraints, that's happening all the time.
Yeah in theory someone could map with a different starting trajectory, but for us it's not the case (for now)